build/common.sh

```diff
@@ -92,6 +92,7 @@ kube::build::get_docker_wrapped_binaries() {
     "amd64")
         local targets=(
           cloud-controller-manager,busybox
+          hyperkube,busybox
```

Comment:
hyperkube has its own base image, and is built using this function: https://github.com/kubernetes/kubernetes/blob/a0ed61671cb86f26dbce128be857a95a3d33b5ec/build/lib/release.sh#L257-L275.

We also don't want to list it here, since these are the images included in kubernetes-server.tar.gz, and I don't think we want to include hyperkube in it.

Building hyperkube also used to be very slow, but that is no longer true. Flipping the default of [`KUBE_BUILD_HYPERKUBE`](https://github.com/kubernetes/kubernetes/blob/a0ed61671cb86f26dbce128be857a95a3d33b5ec/build/lib/release.sh#L31) would be fine to me.

Reply:
ah KUBE_BUILD_HYPERKUBE sounds good. reworked!